docs/glossary.md

```diff
@@ -0,0 +1,184 @@
+---
+id: glossary
+title: Glossary
+sidebar_label: Glossary
+description:  Avail common reference for general definitions and concepts.
+
+keywords:
+    - docs
+    - avail
+    - modularity
+    - Monolithic
+    - data availability
+image: https://availproject.github.io/img/avail/AvailDocs.png
+
+
+
+---
+
+# Avail Glossary
+## General
+
+### Modular Blockchain
+A modular blockchain is one that focuses on handling a select few duties and outsources the rest to one or more separate layers.
+
+### Monolithic 
+In a monolithic chain, all tasks are handled on a single layer or a group of tightly coupled chains operating on the same layer. The latter distinction is important to note: a network of interconnected blockchains that handle all roles, such as Polkadot’s parachains or Avalanche’s subnets, doesn’t qualify as modular architecture.
+
+### Data Availability
+Data availability is the guarantee that block proposers publish all transaction data of a block, along with making this data available to all network participants. In essence, it refers to all nodes’ ability to access and download all the data contained within the various blocks of a network.
+
+### Scalability
+Blockchain scalability refers to a blockchain's capacity to execute transactions, store data, and come to consensus as the network's user base grows.
+
+### Execution
+Execution is how nodes on the blockchain process transactions to transition the blockchain between states. Nodes participating in consensus must execute transactions using their copy of the blockchain to attest before validating blocks.
+
+### Settlement
+The "finality" that blockchains offer is a promise that the transactions that have been recorded in the chain's history are unchangeable (or "immutable"). The blockchain must be persuaded of a transaction's authenticity for this to occur. The chain must therefore validate transactions, check the accuracy of proofs, and resolve disagreements in order to perform the settlement function.
```

Comment:
This is not correct (or at best confusing) in Avail/modular blockchain terms.

Finality refers to when blocks (and transactions/data therein) become irreversible. Settlement is something else.

Since Avail is a modular base chain, it does not execute transactions posted as raw data to it. Therefore, this data can be anything—even invalid data / spam. Settlement refers to the process by which modular layers agree on the correct execution outcome of the transaction data, including any dispute resolution process necessary.

For example, a validium "settles" on Ethereum—transaction data is published to Avail, one or more sequencers execute transactions, and proofs are posted to Ethereum for settlement. Other constructions can have different settlement mechanisms / places where settlement occurs.